Rbac tests created

cypo721 opened the pull request:
Finished all rbac test cases for standalone  and domain mode.

Commit:
Created standalone RBAC testcase, not finished all tests.

Commit:
Fineshed RBAC  test for standalone mode.

Created Administrator/SuperUser- Group/User test and RuntimeAccessError test.

Commit:
Updated FinderNavigation with method which wait for first column of navigation.

Created method waitForFirstNavigationPanel.

Commit:
Added method for clearing path of addresses in FinderNavigation.

Commit:
First part of RBAC domian test cases migrated.

Created DomainRuntimeAccessError test, Profile/ServerAvailability test,
ScopeRole test and updated ManagedDeployment test.

Commit:
Fixed role assignment test case in standalone mode.

Commit:
Group and host scoped role test cases created.
Finished group and host scoped role tests.

Commit:
Created server group and server config test cases.

Commit:
Created ServerManagement and DomainElementaryAccessControl test cases.

Created new tests and fixed ElemantaryAccessControl test case.
Small code review (deleted unused methods, imports, comments).
All RBAC test finished.

zebrik commented on common/src/main/java/org/jboss/hal/testsuite/util/Console.java:
```diff
@@ -287,6 +287,16 @@ public void pageDown() {
         return this;
     }
 
+    public <T extends BasePage> Console waitForFirstNavigationPanel(Class<T> clazz) {
```
Could this method call `refreshAndNavigate` and then wait? To avoid code duplication and increase readability.

cypo721 replied:
Method updated.

zebrik commented on common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java:
```diff
@@ -202,6 +204,10 @@ public FinderNavigation resetNavigation() {
         return this;
     }
 
+    public void clearNavigation() {
```
Please add javadoc for this method to emphasize the difference compared to `resetNavigation` method and when to use this method.

cypo721 replied:
Javadoc added.

zebrik commented:
Apart from two details (see comments above) LGTM.

Commit:
Updated Console and FinderNavigation.

Updated javadoc for clearNavigation method in FinderNavigation.
Updated waitForFirstNavigationPanel method in Console.

zebrik commented:
Thank you!
